scripts/version-sync.sh

```diff
@@ -116,6 +116,16 @@ for i in "${!FILES_TO_CHECK[@]}"; do
     else
         # Replace semver in VERSIONFILE with semver obtained from SOURCE_FILE
         TMPFILE=$(mktemp /tmp/new_version.XXXXXX)
+	# Check sed version, exit if version < 4.3
+	if ! /usr/bin/sed --version > /dev/null 2>&1; then
```

Comment:
```suggestion
	if ! sed --version > /dev/null 2>&1; then
```
apologies, i added the `/user/bin` in my prev comment which was left over from testing. we should just use the sed on the $PATH.